src/codegen/property_body.rs

```diff
@@ -131,7 +139,10 @@ impl Builder {
         let unwrap = if self.is_nullable {
             // This one is strictly speaking nullable, but
             // we represent that with an empty Vec instead
-            if self.type_ == "Vec<GString>" {
+            if ["Vec<GString>", "Vec<crate::GString>", "Vec<glib::GString>"]
```

Comment:
Better check than the `ends_with()` you have elsewhere. Sounds like this could use a helper function `is_gstring()` :)